herwig.sh

```diff
@@ -31,27 +32,15 @@ export LDFLAGS="-L$LHAPDF_ROOT/lib -L$CGAL_ROOT/lib -L$GMP_ROOT/lib $LDZLIB"
 make ${JOBS:+-j $JOBS}
 make install
 
-# Modulefile
-MODULEDIR="$INSTALLROOT/etc/modulefiles"
-MODULEFILE="$MODULEDIR/$PKGNAME"
-mkdir -p "$MODULEDIR"
-cat > "$MODULEFILE" <<EoF
-#%Module1.0
-proc ModulesHelp { } {
-  global version
-  puts stderr "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
-}
-set version $PKGVERSION-@@PKGREVISION@$PKGHASH@@
-module-whatis "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
-# Dependencies
-module load BASE/1.0 ThePEG/$THEPEG_VERSION-$THEPEG_REVISION \\
-            ${LHAPDF_PDFSETS_REVISION:+lhapdf-pdfsets/$LHAPDF_PDFSETS_VERSION-$LHAPDF_PDFSETS_REVISION} \\
-            ${OPENLOOPS_REVISION:+Openloops/$OPENLOOPS_VERSION-$OPENLOOPS_REVISION}                     \\
-            ${MADGRAPH_REVISION:+madgraph/$MADGRAPH_VERSION-$MADGRAPH_REVISION}
+#ModuleFile
+mkdir -p etc/modulefiles
+alibuild-generate-module > etc/modulefiles/$PKGNAME
+cat >> etc/modulefiles/$PKGNAME <<EoF
 # Our environment
 set HERWIG_ROOT \$::env(BASEDIR)/$PKGNAME/\$version
 setenv HERWIG_ROOT \$HERWIG_ROOT
 setenv HERWIG_INSTALL_PATH \$::env(HERWIG_ROOT)/lib/Herwig
 prepend-path PATH \$HERWIG_ROOT/bin
-prepend-path LD_LIBRARY_PATH \$HERWIG_ROOT/lib/Herwig
+prepend-path LD_LIBRARY_PATH \$HERWIG_ROOT/lib/Herwigb
```

Comment:
Maybe my ignorance of alibuild-recipe-tools.sh, but ... is this "b" at the end of the line "../lib/Herwigb" added on purpose ?
Same is done apparently for OpenLoops.sh

Reply:
@maireiphc Thanks for spotting! As you can imagine this is not a feature of alibuild-recipe-tools but a simple typo, not sure how it entered. I will fix immediately!